hknweb/candidate/templates/candidate/offreq.html

```diff
@@ -36,12 +37,21 @@ <h3>Past Officer Challenges</h3>
                         </td>
                         <td align="center">{{ challenge.requester }}</td>
                         <td align="center">
-                            {% if not challenge.reviewed %}
-                                <img src="{% static 'candidate/maybe.png'%}" alt="in progress" class="table-icon">
-                            {% elif challenge.confirmed %}
+                            {% if challenge.officer_confirmed is False or challenge.csec_confirmed is False %}
```

Comment:
If we're using this kind of conditional logic repeatedly, maybe we should have methods on the OffChallenge object for confirmed() or declined() or in_progress().

Reply:
Can you access those methods from Django templates

Reply:
I used a property instead! Unfortunately I can't use properties to filter querysets, so index.html still duplicates code